src/sentry/models/release_threshold/constants.py

```diff
@@ -41,32 +41,24 @@ def as_str_choices(cls):
 
 
 class TriggerType:
-    PERCENT_OVER = 0
-    PERCENT_UNDER = 1
-    ABSOLUTE_OVER = 2
-    ABSOLUTE_UNDER = 3
+    OVER = 0
+    UNDER = 1
 
-    PERCENT_OVER_STR = "percent_over"
-    PERCENT_UNDER_STR = "percent_under"
-    ABSOLUTE_OVER_STR = "absolute_over"
-    ABSOLUTE_UNDER_STR = "absolute_under"
+    OVER_STR = "over"
```

Comment:
Why do you need this to be both int and str?

Reply:
It's an enum which is generally just safer for DB fields.
If these ever change/update, we won't need to chase down all the strings everywhere.